force-app/main/default/classes/PharmanetPayloadTest.cls

```diff
@@ -18,21 +17,12 @@ public with sharing class PharmanetPayloadTest {
         system.assertEquals('1111', saApprovalRequests[0].saRecord.specialItem.din);
     }
 
-    private static StepAction newPharmanetPayload() {
-        return new StepAction(
-            StepAction.ActionType.PHARMANET_PAYLOAD,
-            null,
-            null,
-            null,
-            '1111',
-            'DIN',
-            'B',
-            100,
-            10,
-            null, 
-            'Indefinite',
-            null,
-            null
-        );
+    
+    
+    Private static Step_Action__c getStepActionRecord(){
```

Comment:
Access modifiers in Salesforce (https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_access_modifiers.htm) are lowercase. There is so much inconsistency in the code because the old methods in this apex class have lowercase and this new getStepActionRecord is in upper-case. If it was typo, please fix it.

Reply:
Thanks @cgiSunilM  for finding out . Corrected the typo from our end.